Formula/httpd.rb

```diff
@@ -0,0 +1,132 @@
+class Httpd < Formula
+  desc "HTTP server"
+  homepage "https://httpd.apache.org/"
+  url "https://archive.apache.org/dist/httpd/httpd-2.4.27.tar.bz2"
```

Comment:
The url should be
```
  url "https://www.apache.org/dyn/closer.cgi?path=httpd/httpd-2.4.27.tar.bz2"
```

and if you want a mirror (though it's usually pointless) then
```
  mirror "https://archive.apache.org/dist/httpd/httpd-2.4.27.tar.bz2"
```